widget-reference/table.md

```diff
@@ -63,3 +63,4 @@ Tables are useful to view large lists of data. To drill down into the data of a
 | **onRowSelected** | Sets the action to be run when user selects a row in the table. Default supported actions are: Call API, Execute DB Query, Navigate to Page, Show Alert, Open / Close popup |
```

Comment:
@gogetter22 can you update the gif to drop the table, resize it so it's more visible, and bind it to a query.

Reply:
@gogetter22 something seems wrong with the table in the gif, the bg is transparent instead of white. How did this happen?